dace/sdfg/graph.py

```diff
@@ -664,10 +664,13 @@ def add_edge(self, src: NodeT, dst: NodeT, data: EdgeT = None):
         return self._nx.add_edge(src, dst, data=data)
 
     def remove_node(self, node: NodeT):
-        for edge in itertools.chain(self.in_edges(node), self.out_edges(node)):
-            self.remove_edge(edge)
-        del self._nodes[node]
-        self._nx.remove_node(node)
+        try:
+            for edge in itertools.chain(self.in_edges(node), self.out_edges(node)):
+                self.remove_edge(edge)
+            del self._nodes[node]
+            self._nx.remove_node(node)
+        except KeyError:
+            pass
```

Comment:
Ignored deliberately or missed?

Reply:
Removing a node that is already not in the graph now won't complain _and_ run in decent performance (otherwise need to look it up every time).

Reply:
It matches the behavior of networkx iirc